#163019081 Configure project to use es6 or later version

Easybuoy opened the pull request:
#### What does this PR do?
Configure project to use es6 or later version.

#### Description of Task to be completed?
- Install <code>babel dependencies </code> for transpiling.
- Create <code>.babelrc</code> in the root of the project.
- Add <code>babel preset</code> to <code>.babelrc</code> file.

#### How should this be manually tested?
- Use <code>es6</code> features like <code>import</code> instead of <code>require</code> in any file and verify <code>server</code> runs successfully.

### What are the relevant pivotal tracker stories?
#163019081

### Screenshots (if appropriate)
None

#### Any background context you want to provide?
None

Easybuoy committed:
chore(api) configure project to use es6 or later
- created and added preset to .babelrc
install babel dependencies
[Delivers #163019081]

oreoluwade commented:
@Easybuoy You need to fix the eslint violations/warnings.

Remove unused imports and so on.

Also, I can see an error/warning in the console when I start the server. I'd like you to fix that as well. Let me know if you need further clarifications.

Easybuoy committed:
chore(api) configure project to use es6 or later
- created and added preset to .babelrc
install babel dependencies
[Delivers #163019081]

oreoluwade commented on package.json:
```diff
@@ -30,6 +31,11 @@
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.5",
```
What is the function of the `babel-loader` dependency?

Easybuoy replied:
<code>babel-loader</code> is used in transpiling javascript files using babel and webpack. Basically for transpiling webpack files.

oreoluwade replied:
Yeah, but we aren't using webpack at this time.

Easybuoy replied:
okay, i will remove and push changes.

Easybuoy replied:
I have pushed the change.

Easybuoy committed:
chore(api) configure project to use es6 or later
- created and added preset to .babelrc
install babel dependencies
[Delivers #163019081]

Easybuoy commented:
> @Easybuoy You need to fix the eslint violations/warnings.
> 
> Remove unused imports and so on.
> 
> Also, I can see an error/warning in the console when I start the server. I'd like you to fix that as well. Let me know if you need further clarifications.

I've fixed this.

Easybuoy committed:
chore(api) configure project to use es6 or later
- removed babel-loader from dependency
[Delivers #163019081]
